packages/cli/src/cli/commands/checkCommand.js

```diff
@@ -5,10 +5,16 @@ const checkCommand = new Command('check')
 checkCommand.description('Checks given JS files for trace occurencies')
 checkCommand.arguments('<pattern>')
 checkCommand.option('-t, --trace [trace]', 'the name of namespace containing strings')
+checkCommand.option(
+  '-p, --plugin <names...>',
+  'the names of the plugins that contain the checks to run'
+)
```

Comment:
I don't know how to pass plugin options using that syntax. Well, "plugin" can be a name of npm package, but it is not clear how to pass options to plugin.

I am leaning towards removing `--plugin` option and instead add `-c, --config` option which will allow to specify configuration file. [cosmiconfig](https://www.npmjs.com/package/cosmiconfig) is a popular npm package which can be used for handling various formats of configuration files. We can use it in tandem with `--config` option, that is, cli will read `bv.config.js` or `.bvrc.json` by default and it will be possible specify config file via `bundle-validator --config ./some_dir/bv.config.js`.

Can you work on adding `cosmiconfig` in scope of this MR and console.log the resulting config in `./lib/check.js` similar to how you log the list of files?